### ISSUE TITLE
Golbat causes The Pillar to activate out of order

### ISSUE THREAD

**Issue by wingedcatgirl** [InertSteak/Pokermon#10]
A card whose enhancement is removed by Golbat will become debuffed by The Pillar before it is scored, negating both the increase to Golbat's attributes and any other effects.

**Comment by InertSteak** [comment]
Yeah that's weird, I'm not sure what's going on there but I'll try to look into it

**Comment by PokemasterLord** [comment]
I noticed this with other jokers that modify the card as well, such as Golduck, the card will change and get debuffed before it scores.

**Comment by InertSteak** [comment]
Good news, this has been fixed since steamodded reworked their calc engine!
